src/services/lease-service/adapters/xpand/estate-code-adapter.ts

```diff
@@ -0,0 +1,44 @@
+import Config from '../../../../common/config'
+import knex from 'knex'
+
+const db = knex({
+  client: 'mssql',
+  connection: Config.xpandDatabase,
+})
```

Comment:
Jag _tror_ det är en dålig idé att ha flera separata connections till samma db på det här viset. Eventuellt har vi redan det vilket isf är dåligt och ingen idé att introducera fler. 

Kanske extrahera till xpand-db.ts eller något. Jag gjorde liknande för leasing.

Reply:
Just nu finns det 3 möjliga connections till xpand. Jag är rätt osäker på livscykeln för dessa connections för att vara ärlig. Det bästa är ju som du säger att skapa en connection och passa in samma connection till tjänsterna som använder den. 

Det känns dock som en nytt kort? Jag kan lägga till det i backlogen och så får vi kollar nogare på det?

Reply:
Tror inte du behöver passain. Räcker att lägga 
```

const db = knex({
  client: 'mssql',
  connection: Config.xpandDatabase,
})
```
i en egen fil och sen importera

Reply:
Men sure, vi kan göra ett kort på det.
Vet inte heller hur underliggande ser ut exakt men med flera separata connections kan man inte garantera databasens integritet. T.ex kan två olika transactions köras samtidigt som påverkar samma tabeller.